src/components/Records/Search/Output/SearchOutput.vue

```diff
@@ -15,7 +15,10 @@
     />
 
     <!--List Row-->
-    <div :class="['opacity-0-transition',{'opacity-1-transition':!isColumnList}]">
+    <div
+      v-if="records.length"
```

Comment:
why not v-if="loading" ?

Reply:
cz this if is about checking whether there is no data available not related to loading data

Reply:
OK i understand.
I think making a small computed property that either returns null or the records would make more sense in terms of logic. You can then do v-if='!records' for instance instead of testing for an existing length then testing for a length inferior to 1.
Alternativaly, we can make a getter in the records store that would do the same thing: return the results of the search, or null.